Sources/VitaminUIKit/Components/Snackbar/VitaminSnackbar.swift

```diff
@@ -286,9 +286,10 @@ extension VitaminSnackbar {
     }
 
     private func updateMessageLabelPreferredMaxWidth() {
-        if image != nil {
-            let margins = rightMargin + imageWidthAndHeight + imageLeftAndRightMargin
+        guard image == nil else {
```

Comment:
I'm not sure about the usage of `guard`. It's maybe better to keep `guard` only for dependency checks and not for the condition on a feature 🤔 But it's an open debate 😅

Reply:
While using an ìf`, I forgot to write my `return` statement (or `else`), leading to the issue.
And that would have not been possible with this `guard` syntax, that is why I switched.
But ok to switch back to `if`;)

Reply:
Yes, I didn't think about this use case 🤔